soh/soh/Enhancements/randomizer/hook_handlers.cpp

```diff
@@ -193,7 +193,7 @@ void RandomizerOnFlagSetHandler(int16_t flagType, int16_t flag) {
     auto loc = Rando::Context::GetInstance()->GetItemLocation(rc);
     if (loc == nullptr || loc->HasObtained()) return;
 
-    SPDLOG_INFO("Queuing RC: {}", rc);
+    SPDLOG_INFO("Queuing RC: {}", static_cast<uint32_t>(rc));
```

Comment:
Where did these `static_cast` come from? Seems like we can keep them though.

Reply:
Was giving me an error of "no valid type" without it.